src/RevCPU.cc

```diff
@@ -861,7 +861,7 @@ void RevCPU::HandleThreadStateChangesForProc( uint32_t ProcID ) {
 
 void RevCPU::InitMainThread( uint32_t MainThreadID, const uint64_t StartAddr ) {
   // @Lee: Is there a better way to get the feature info?
-  std::unique_ptr<RevRegFile> MainThreadRegState = std::make_unique<RevRegFile>( Procs[0]->GetRevFeature() );
+  std::unique_ptr<RevRegFile> MainThreadRegState = std::make_unique<RevRegFile>( Procs[0] );
```

Comment:
The `RevRegFile` constructor has been changed to take the parent `RevCore`, from which it can get the `RevFeature` and other fields.